src/gas.ts

```diff
@@ -0,0 +1,62 @@
+import { GasOptions } from './types'
+import { JsonRpcProvider, ethers, getBigInt } from 'ethers'
+
+
+export const getDefaultSameChainGas = (): GasOptions => {
+  return {
+    maxFeePerGas: "0",
+    maxPriorityFeePerGas: "0",
+    verificationGasLimit: "1500000",
+    callGasLimit: "1500000",
+    preVerificationGas: "90000"
+  }
+}
```

Comment:
preverification: 50000, verification: 120000 and as discussed the destination userOp needs network specific maxFeePerGas, maxPriorityFeePerGas

Reply:
saw your note below about the maxFeePerGas/MaxPriorityFeePerGas, but adjust the other values.